docs/source/configuration.md

```diff
@@ -126,5 +126,17 @@ clients using these URLs. By default, the JWT auth provider is used here.
 
 There is typically no need to override the default behavior.
 
+#### `LEGACY_ENDPOINTS`
+This is a `bool` flag, default `true` (deprecated, use `false` where possible), that affects the base URI of all the service endpoints. Previously, the endpoints didn't adhere to the rules for [automatic LFS server discovery](https://github.com/git-lfs/git-lfs/blob/main/docs/api/server-discovery.md), which needed additional routing or client configuration.
+
+The default base URI for all giftless endpoints is now `/<org_path>/<repo>.git/info/lfs` while the legacy one is `/<org>/<repo>`.
```

Comment:
Cool.  It may take me personally some time to convert all our repos, but having discovery work is awesome.